Makefile

```diff
@@ -159,7 +161,7 @@ clean:
 compose-build:
 	@echo "Tearing down the docker compose images, network, etc..."
 	@docker-compose down --remove-orphans
-	@docker-compose build
+	@docker-compose build --build-arg REQUIRE_MSSQL="true"
```

Comment:
why is this hard-coded to `true`? Particularly, not sure if `pytest-integration-external` needs mssql deps

Reply:
When we run Fidesops in production, we'll always want to install the MSSQL deps so that a user can connect an MSSQL instance if they wish. This `compose-build` task is used by the `Makefile`, and not the `run_infrastructure.py` script that `make pytest-integration-external` will eventually use.

Good spot though, we don't need this here in that instance!